eth-connector/src/lib.rs

```diff
@@ -166,10 +120,10 @@ impl EthConnectorContract {
             amount,
         );
 
-        // Verify message data before `ft_on_transfer` call for Engine accounts
+        // Verify message data before `ft_on_transfer` call for Engine account
         // to avoid verification panics inside `ft_on_transfer`.
-        // Allowed empty message if `receiver_id != known_engin_accounts`.
-        if self.known_engine_accounts.contains(&receiver_id) {
+        // Allowed empty message if `receiver_id != aurora_engine_account_id`.
+        if self.aurora_engine_account_id == receiver_id {
```

Comment:
There was some idea why there could be many of them, not just Aurora. But I don't remember that idea anymore.

Reply:
Yes, it was for silos, but we can ignore that